demos/python_demos/text_spotting_demo/README.md

```diff
@@ -68,10 +68,10 @@ Options:
                         recognition model (decoder part).
   -i "<path>"           Required. Input to process.
   -d "<device>", --device "<device>"
-                        Optional. Specify the target device to infer on: CPU,
-                        GPU, FPGA, HDDL or MYRIAD. The demo will look for a
-                        suitable plugin for device specified (by default, it
-                        is CPU).
+                        Optional. Specify the target device to infer on, i.e. CPU, GPU.
+                        The demo will look for a suitable plugin for device specified
+                        (by default, it is CPU). Please refer to OpenVINO documentation
+                        for the list of devices supported by the model.
```

Comment:
You need to change the help text in the code accordingly.

Reply:
yes, fixed as well

Reply:
You didn't change it in the same way. The text needs to be identical (in fact, you're supposed to just run the demo with `--help` and paste the output into the README).

Reply:
that's right, corrected